src/dialer.js

```diff
@@ -1,43 +1,48 @@
 const sessions = {};
 let currentSession;
-let inConference = false;
+const inConference = false;
 let currentAtxfer;
+let countDown;
+let timerId;
 // let sessionIdsInMerge = [];
 
-const setFullName = async () => {
-  const user = await Wazo.getApiClient().confd.getUser(Wazo.Auth.getSession().uuid);
-  const fullName = user.firstName ? `${user.firstName} ${user.lastName}` : user.username;
+const setGreeter = async () => {
+  const user = await Wazo.getApiClient().confd.getUser(
+    Wazo.Auth.getSession().uuid,
+  );
+  const name = user.firstName;
 
-  $('#user').html(`${fullName} (${user.lines[0].extensions[0].exten})`);
+  $('.greeter').html(`Hello ${name} 👋`);
 };
 
-const setMainStatus = status => {
-  $('#dialer .status').html(status);
+const setMainStatus = (status) => {
+  $('#status').html(status);
+  setTimeout(() => $('#status').html('&nbsp;'), 3000);
 };
 
-const getNumber = callSession => callSession.realDisplayName || callSession.displayName || callSession.number;
+const getNumber = (callSession) => callSession.realDisplayName || callSession.displayName || callSession.number;
 
-const getStatus = callSession => {
+const getStatus = (callSession) => {
   const number = getNumber(callSession);
 
   if (callSession.paused) {
-    return 'Call with ' + number + ' on hold';
+    return `Call with ${number} on hold`;
   }
 
   if (callSession.muted) {
-    return 'Call with ' + number + ' muted';
+    return `Call with ${number} muted`;
   }
 
   switch (callSession.sipStatus) {
     case Wazo.Phone.SessionState.Initial:
     case Wazo.Phone.SessionState.Establishing:
-      return 'Calling ' + number + '...';
+      return `Calling ${number}...`;
     case Wazo.Phone.SessionState.Established:
-      return 'On call with : ' + number;
+      return `On call with : ${number}`;
     case Wazo.Phone.SessionState.Terminated:
-      return 'Call canceled by ' + number;
+      return `Call canceled by ${number}`;
     default:
-      return 'Unknown status: ' + callSession.sipStatus;
+      return `Unknown status: ${callSession.sipStatus}`;
```

Comment:
`Call in ${callSession.sipStatus} status` because `Unknown` can be seen as messy.